source/collaborate/invite-people.rst

```diff
@@ -48,7 +48,18 @@ Direct invites are invitation emails sent from your team's server directly to th
 .. tab:: Mobile
 
   1. Tap the |plus| icon in the top right corner of the screen.
+
+    .. image:: ../images/direct-mobile-invite-options.jpg
```

Comment:
@Rutam21 - Please scale the new images by adding ``:scale: 60`` on the line after ``:alt:``. Let's start with scaling to 60% and see how they display on the page.

Reply:
This fix is applied to all images in this PR.